modules/restarting-installation.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// installing/installing-troubleshooting.adoc
+
+[id="restarting-installation_{context}"]
+= Reinstalling the {product-title} cluster
+
+If you are unable to debug and resolve issues in the failed {product-title} installation, you might need to consider installing a new {product-title} cluster. Before starting the installation process again, you must do a thorough cleanup.
+
+.Procedure
+
+1. Destroy the cluster and remove all the resources associated with the cluster, including the hidden installer state files in the installation directory:
++
+[source, terminal]
+----
+$ ./openshift-install destroy cluster --dir=<installation_directory> <1>
```

Comment:
This command is only applied on IPI cluster.  For UPI cluster, user need to cleanup the cluster and associated resources by themselves based on how they create the infrastructure.